kie-soup-dataset/kie-soup-dataset-sql/src/main/java/org/dashbuilder/dataprovider/sql/JDBCUtils.java

```diff
@@ -114,12 +114,14 @@ public static void execute(Connection connection, String sql) throws SQLExceptio
         }
     }
 
-    public static ResultSet executeQuery(Connection connection, String sql) throws SQLException {
+    public static ResultSetHandler executeQuery(Connection connection, String sql) throws SQLException {
         try {
             if (log.isDebugEnabled()) {
                 log.debug(sql);
             }
-            return connection.createStatement().executeQuery(sql);
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery(sql);
```

Comment:
@jesuino FYI: Sonarcloud yells this is a problem (we are not closing the statement), but imho it is false positive in this particular case so I marked it as won't fix for the future.